salt/client/mixins.py

```diff
@@ -293,10 +293,7 @@ def low(self, fun, low):
                     continue
                 completed_funcs.append(mod)
                 for global_key, value in six.iteritems(func_globals):
-                    if six.PY3:
-                        self.functions[fun].__globals__[global_key] = value
-                    else:
-                        self.functions[fun].func_globals[global_key] = value  # pylint: disable=incompatible-py3-code
+                    self.functions[fun].__globals__[global_key] = value
```

Comment:
This is not Py3 compatible... I was compatible how it was.

PR's don't check Py3 compat yet.